backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -358,29 +323,23 @@ export class TransactionResolver {
       throw new Error('invalid recipient public key')
     }
 
-    // validate amount
-    if (amount <= 0) {
-      throw new Error('invalid amount')
-    }
-
     const centAmount = Math.trunc(amount * 10000)
 
     const queryRunner = getConnection().createQueryRunner()
     await queryRunner.connect()
     await queryRunner.startTransaction('READ UNCOMMITTED')
     try {
       // transaction
-      let transaction = new dbTransaction()
+      const transaction = new dbTransaction()
       transaction.transactionTypeId = TransactionTypeId.SEND
       transaction.memo = memo
+      transaction.userId = senderUser.id
+      transaction.pubkey = senderUser.pubKey
+      transaction.sendReceiverUserId = recipientUser.id
```

Comment:
The model should change here. `sendReceiverUserId` is misleading. `sendRecipientUserId`